### PR TITLE
Walking directions

### PR DESCRIPTION
**Pull request by bgregg**
For a stop map. Don't merge yet, this only gives a path, not text directions. I will continue to work and add text directions very soon.

### CODE REVIEW

**Comment by akaplo** [comment]
Just zap this PR to remind me about it when it's ready for merging.

**Comment by bgregg** [comment]
This is pretty much where I want it. Look through it and let me know what you think

**Comment by bgregg** [comment]
screen shot 2016-03-31 at 3 13 05 pm

Here's what it looks like on the web app. Not ideal, but I am having some trouble fixing it. Maybe for another release since our main focus here is the mobile app

**Comment by bgregg** [comment]
screen shot 2016-03-31 at 3 14 57 pm

But here's what the final version looks like on mobile. Pretty slick huh?

File: .gitignore [hunk 1]
```diff
@@ -42,3 +42,6 @@ jspm_packages
 
 # Optional npm cache directory
 .npm
+
+#those darn DS_Store files
+**/.DS_Store
```
Comment (akaplo) [inline .gitignore:47]: ~~Could've sworn I'd added those already!~~ They must be in my [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).

**Comment by akaplo** [comment]
This looks **fantastic.**  I have no line notes but two general comments:
1. If you haven't already, run the linter with `gulp lint-n-fix` and then do a sweep to make sure it didn't accidentally break anything (which seems to have happened on #110)
2. Since we've added the callback functionality to the `Map.plotCurrentLocation()` method [here](https://github.com/umts/pvta-multiplatform/pull/100/files#diff-86c7c39f5b7930f8f1382e08f6e5002eR306), have you gone through every page that uses currentLocation and made sure that it doesn't break them? I can't see why it would, since it's technically optional, but you never know.

Great work!

**Comment by bgregg** [comment]
I just looked through it and it didnt break the plotCurrentLocation() method. Everywhere else that it uses a map it works. 

This pretty much closes this one, unless you have something else, @akaplo? 

Closes #100.

**Comment by akaplo** [comment]
Great!

**Comment by akaplo** [comment]
Merge away

**Comment by bgregg** [comment]
Merged #100.

**Pull request merged**